### PR TITLE
Fixes learning guide order and iReading calendar popup student count

### PR DESCRIPTION
**Pull request by jpslav**

### CODE REVIEW

**Comment by philschatz** [comment]
:+1: Works for me!

File: app/routines/get_course_stats.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@ def exec(role:, course:)
     run(:get_course_books, course: course)
     run(:visit_book, book: outputs.books.first, visitor_names: [:toc, :page_data])
 
-    chapters = compile_chapters
+    chapters = compile_chapters.sort_by{|ch| ch[:chapter_section]}
```
Comment (kevinburleigh75) [inline app/routines/get_course_stats.rb]: It's not obvious to me that page[:chapter_section] is sortable in the way we want - will 10.3 come before 2.4?

File: app/routines/get_course_stats.rb [hunk 2]
```diff
@@ -41,7 +41,7 @@ def compile_chapters
         task_type: :mixed_practice
       )
 
-      pages = compile_pages(task_steps: task_steps)
+      pages = compile_pages(task_steps: task_steps).sort_by{|page| page[:chapter_section]}
```
Comment (kevinburleigh75) [inline app/routines/get_course_stats.rb]: Same question as L24.

File: app/routines/get_course_stats.rb (hunk 1 above)
Comment (jpslav) [inline app/routines/get_course_stats.rb]: I admit that I'm not entirely sure it is 100% right, but it works enough for now.  Longer strings are sorted after shorter strings, and shorter strings are compared by characters, so I **think** it is ok.

**Pull request merged**